data/data/agent/systemd/units/agent-auth-token-status.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Service that checks if agent auth token is unexpired and displays a message if expired.
+Wants=network-online.target
+After=network-online.target agent-interactive-console.service
+ConditionPathExists=/etc/assisted/add-nodes.env
+
+[Service]
+Type=simple
+EnvironmentFile=/etc/assisted/add-nodes.env
```

Comment:
To make the service run on day1 and day2, the env file needs to be changed to a common file for both days,

Reply:
We only need to run it on day2 which is fulfilled with add-nodes.env